src/searchIssues.js

```diff
@@ -64,7 +64,7 @@ function searchBySummary(baseFileDir, source, params, custom_filter_str) {
 }
 
 module.exports = (baseFileDir, source, params, callback) => {
-  const searchUrl = source.url + '/rest/api/2/search/';
+  const searchUrl = source.url + '/rest/api/3/search';
```

Comment:
@RakesPati `/rest/api/3/search` is also deprecated. Per <https://developer.atlassian.com/changelog/#CHANGE-2046>, we should instead be migrating to [`/rest/api/3/search/jql`](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-search/#api-rest-api-3-search-jql-get).